backends/ebpf/psa/ebpfPsaControl.h

```diff
@@ -71,7 +74,9 @@ class EBPFControlPSA : public EBPFControl {
 
     EBPFControlPSA(const EBPFProgram *program, const IR::ControlBlock *control,
                    const IR::Parameter *parserHeaders)
-        : EBPFControl(program, control, parserHeaders) {}
+        : EBPFControl(program, control, parserHeaders),
```

Comment:
can you add the skb_metadata in EBPFControlPNA class which is in TC

Reply:
I'll pass this on to mouse

Reply:
Was able to pass it to `EBPFControlPNA`